go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/onsi/gomega v1.30.0
 	github.com/prometheus/client_golang v1.17.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/crypto v0.16.0
```

Comment:
Why the deps bump?

Reply:
probably by running some `go mod tidy ` .. will remove